tests/FakeItEasy.IntegrationTests/ExternalAssemblyGenerator.cs

```diff
@@ -27,10 +29,16 @@ public ExternalAssemblyGenerator()
 
         /// <summary>
         /// Gets the path to a copy of <see cref="AssemblyOriginalPath"/>.
-        ///
+        /// </summary>
         public string AssemblyCopyPath { get; }
 
+        /// <summary>
+        /// Gets the path of an assembly on which a type of FakeItEasy.IntegrationTests.External depends.
+        /// </summary>
+        public string AssemblyDependencyPath { get; }
+
         private const string AssemblyName = "FakeItEasy.ExtensionPoints.External";
+        private const string DependencyAssemblyName = "FakeItEasy.ExtensionPoints.ExternalDependency";

```

Comment:
This assembly is referenced by `FakeItEasy.ExtensionPoints.External`, but won't be available at run time. This is done in order to cause a type load error when we scan the types in `FakeItEasy.ExtensionPoints.External`.